kodilla-sudoku/src/main/java/com/kodilla/sudoku/validator/SudokuValidator.java

```diff
@@ -11,7 +11,6 @@ public SudokuValidator(SudokuBoard board) {
     }
 
     public boolean validate(SudokuElement element) {
```

Comment:
Ta metoda nie waliduje przecież. Czy to w ogóle jest walidator? Może lepiej nazwać tę klasę checker?